src/vis.ts

```diff
@@ -0,0 +1,38 @@
+import * as THREE from 'three';
+import * as plane from './plane';
+import {OrbitControls} from 'three/examples/jsm/controls/OrbitControls';
+
+export class Vis {
+  renderer: THREE.WebGLRenderer;
+  scene: THREE.Scene;
+  camera: THREE.PerspectiveCamera;
+  aircraft: THREE.Group;
```

Comment:
I'm a bit confused on aircraft vs plane -- is plane the singular of aircraft in your code? If so, I'd recommend naming things "plane" and "planes".

I'm saying this bc there's some ambiguity between "plane (vehicle)" and "plane (geometry)", and both could apply when you're rendering aircraft.

Reply:
Renamed to `aircraftModels` to specify function and plural nature. I don't intend to have more than one for a long time (if ever) but it's easy enough to keep the door open.